aerospike.template.conf

```diff
@@ -33,16 +33,22 @@ network {
 	}
 
```

Comment:
i m assuming hyc configurations (dirty, clean nm etc) will be present in this file.

Reply:
Yes they are.